downstairs-types/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2025 Oxide Computer Company
+
+use std::net::{IpAddr, SocketAddr};
+use std::path::PathBuf;
+
+use schemars::JsonSchema;
+use serde::Deserialize;
+
+// Repair API types
+#[derive(Deserialize, JsonSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct Eid {
```

Comment:
Is this needed any more? It looks like it was replaced with `ExtentPath` in `extent_repair_ready` and `get_files_for_extent`?

Reply:
Ah yeah -- would you prefer I rename this back to `Eid` or keep it as `ExtentPath`?

Reply:
Either works, whatever matches our convention - `ExtentPath` has less of a naming collision than `Eid` haha.